tests/integration/targets/nxos_zone_zoneset/tests/common/sanity.yaml

```diff
@@ -144,13 +144,19 @@
       register: result
       cisco.nxos.nxos_zone_zoneset: *id001
 
+    - name: Disaplay Idempotence Check result
+      debug:
+        var: result
+
     - assert:
         that:
-          - result.commands == []
+          - result.commands == ["terminal dont-ask", "zoneset activate name zsetname1
+            vsan 922","zone commit vsan 922", "zoneset activate name zsetname21
+            vsan 923", "no terminal dont-ask"]
 
     - assert:
         that:
-          - result.changed == false
+          - result.changed == true
```

Comment:
@srbharadwaj this task seems to have been idempotent previously. If the changes in this patch inevitably causes the module to loose idempotency in this particular use case, please add a note in the module doc stating the reason for future reference.

Reply:
how to put it in module doc? any example?

Reply:
@NilashishC pls dont merge this yet, we can make this idempotent. i'am working it, will test it out locally and then commit, till then pls keep this PR open